projects/plugins/jetpack/_inc/client/components/tree-dropdown/index.jsx

```diff
@@ -57,8 +57,10 @@ const TreeDropdown = props => {
 		};
 	}, [ handleClickOutside ] );
 
+	const treeDropdownClass = `tree-dropdown ${ hidden ? 'tree-dropdown--hidden' : '' }`;
```

Comment:
Maybe we could use the `classNames` lib already imported on this file.

`import classNames from 'classnames';`